Completed Assignment - 10

ashok-cdw opened the pull request.

ashok-cdw committed:
Completed Assignment - 10

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+
+// 1
+
+cricketPlayers = ["Ashok", "Harish", "Dinesh", "Maamathi", "Kavya", "Ranjith", "Charan", "Hareesh", "Mathi", "Thendral", "Pavithra"]
```
Declarations missing, It should have either var,let,const

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+
+cricketPlayers = ["Ashok", "Harish", "Dinesh", "Maamathi", "Kavya", "Ranjith", "Charan", "Hareesh", "Mathi", "Thendral", "Pavithra"]
+
+console.log(cricketPlayers);
```
It would be better to have consistent way either to have semicolon or not for the entire app, but only stick to one

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+// 5
+
+cricketPlayers.sort(function (a, b) {
+    return a.toLowerCase() - b.toLowerCase();
```
Will this sort ? Review this again in console

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+
+// 6
+
+cricketPlayersWithJerseyNumber = cricketPlayers.map((value) => value + " - " + Math.round(Math.random() * 100))
```
missing declaration

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+
+// 7
+
+cricketPlayersUpperCase = cricketPlayersWithJerseyNumber.map(value => value.toUpperCase())
```
1. missing declaration;
2 . I believe you should use the players instead the playersWithJerseyNumbers

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+// 1 - Generate Numbers from 1 to 100
+
+function generateNumbers(start, end) {
+    let arr = []
```
We only want to display numbers right ? Why do we store them in an array and return them ? but this is nice but not the requirement

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+// Section 3
+
+// 1 - Generate Numbers from 1 to 100
+
```
Can use better commenting instead of single comments use multi line function comments like which are much readable and helps to give insight when you hover on them on their invoke calls for the functions across the app:
/*
@description - This is a function
@params - one,two
*/

csekharb commented on task-10/js/app.js:
```diff
@@ -0,0 +1,105 @@
+
+// 2 - Generate Date in Required Format
+
+function generateDateInFormat(dateObject) {
```
This would give me the next date everytime, say like today is 23/02/2023, it gives me 24/02/2023;
and this looks complex can you simplify this or write a better comment ;
and also why can't you use this instead : https://www.w3schools.com/jsref/jsref_tolocaledatestring.asp

csekharb reviewed (changes requested):
Please review my comments

ashok-cdw committed:
Update app.js

1. added variable declaration to variables
2.corrected mistakes in sort method
3.changed variables
4.added proper commenting to methods
5.removed array and printed directly
6.corrected mistakes in print date method
